### ISSUE TITLE
Child builders for properties not generating correct code

### ISSUE THREAD

**Issue by distantcam** [StefH/FluentBuilder#48]
I'm having a few issues generating builders on a child List property.

#### Scenario 1

```csharp
public class User
{
    public List<Option> Options { get; set; }
}
public class Option
{
    public string Name { get; set; }
}

[AutoGenerateBuilder(typeof(User))]
public partial class MyUserBuilder {}
[AutoGenerateBuilder(typeof(Option))]
public partial class MyOptionBuilder {}
```

Build error:
`...\FluentBuilderGenerator\FluentBuilderGenerator.FluentBuilderSourceGenerator\FluentBuilderSandbox.Option_IListBuilder.g.cs(28,46,28,59): error CS0246: The type or namespace name 'OptionBuilder' could not be found (are you missing a using directive or an assembly reference?)`

#### Scenario 2

Main project

```csharp
public class User
{
    public List<Option> Options { get; set; }
}
public class Option
{
    public string Name { get; set; }
}
```

Test project

```csharp
[AutoGenerateBuilder(typeof(User))]
public partial class UserBuilder { }

[AutoGenerateBuilder(typeof(Option))]
public partial class OptionBuilder { }
```

Build error:
`...\FluentBuilderGenerator\FluentBuilderGenerator.FluentBuilderSourceGenerator\FluentBuilderSandboxTests.UserBuilder.g.cs(31,68,31,86): error CS0234: The type or namespace name 'IListOptionBuilder' does not exist in the namespace 'FluentBuilderSandbox' (are you missing an assembly reference?)`

**Comment by StefH** [comment]
Hello @distantcam, thanks for noticing.

I'll investigate your issue.

**Comment by StefH** [comment]
@distantcam 

Can you please try `FluentBuilder.0.7.0-preview-01` on NuGet ?

**Comment by distantcam** [comment]
Sorry, I missed your earlier comment.

Yep I can confirm this is fixed with version `0.7.0-preview-01`. Thankyou!

**Comment by StefH** [comment]
@distantcam Thank you for validating.

A new 0.7.0 will be released on some time.
